Assets/Mirror/Components/NetworkLobbyManager.cs

```diff
@@ -10,7 +10,7 @@ namespace Mirror
     [HelpURL("https://vis2k.github.io/Mirror/Components/NetworkLobbyManager")]
     public class NetworkLobbyManager : NetworkManager
     {
-        struct PendingPlayer
+        private protected struct PendingPlayer
```

Comment:
isn't 'protected' enough?

Reply:
They were private before.
> private protected
> Access is limited to the containing class or types derived  from the containing class within the current assembly. Available since  C# 7.2.

> protected
> Access is limited to the containing class or types derived from the containing class.

Reply:
don't you think it would be more of an 'easy to use' experience for our users if it's just protected or even public?
the whole goal of mirror is to be the easiest to use networking solution for Unity :)

Reply:
made hem public